docs/flask.asciidoc

```diff
@@ -143,6 +143,18 @@ from elasticapm.contrib.flask import ElasticAPM
 apm = ElasticAPM(app, logging=True)
```

Comment:
`logging=True` seems like a footgun. I'd rather lead with the explanation of `logging=logging.ERROR` and then mention that you can do `logging=True` to get everything incl. the nice warning that you currently have below the `logging=logging.ERROR` example.

It's also worth considering to completely stop mentioning `logging=True`. Setting `logging=logging.DEBUG` has the same effect and is more obviously a footgun.

Reply:
That makes thev docs ten times better! 👍